hieradata/clients/controller.sponsorship.ci.jenkins.io.yaml

```
@@ -392,6 +392,44 @@ profile::jenkinscontroller::jcasc:
           credentialsId: "jenkinsvmagents-userpass"
           usePrivateIP: true
           spot: true
+        - name: "ubuntu-22-inbound-amd64-jdk17"
+          description: "Ubuntu 22.04 LTS"
+          imageDefinition: jenkins-agent-ubuntu-22.04-amd64
+          os: "ubuntu"
+          os_version: "22.04"
+          launcher: "inbound"
+          location: "East US 2"
+          instanceType: Standard_D4ads_v5 # 4 vCPUS / 16 Gb / Max 150 Gb local storage
+          ephemeralOSDisk: true
+          architecture: amd64
+          labels:
+            - ubuntu-22-amd64-maven-17
+          javaHome: /opt/jdk-17 # Test override of the default JDK for builds
```

```suggestion
          javaHome: /opt/jdk-17
```

Comment does not add any valuable information other than the instruction (and it is not a Test ;) )